chipy_org/apps/job_board/models.py

```diff
@@ -41,6 +41,14 @@ def get_queryset(self):
         )
 
 
+class Affiliation(CommonModel):
+    description = models.CharField(max_length=MAX_LENGTH)
```

Comment:
What's the description for here, if it's not what's going to be displayed on the job page? (Currently the viewable description is handled by [this](https://github.com/chicagopython/chipy.org/pull/412/files#diff-b3364321a2ae5d2c87a3fa9356b5a74e64c3eae94254dad103d1d0e5ec5a8b2dR42))

Should the affiliation be tied to some user that has basic auth access to the job creation API endpoint?

Reply:
You can change the status of the job (submitted, approved, rejected) through the admin.  Description is used in the template be default - django tries hard to get a string representation of and object and I had set the __str__ method to return description.  I'll update it though just to make it clear though.

Reply:
Oh duh of course, thanks for clarifying those 😄